doc/ref_model/chapters/chapter04.md

```diff
@@ -263,63 +263,178 @@ Table 4-10 shows performance measurement capabilities.
 
-NOTE: On a Kubernetes based environment, resource allocation will directly depend on how resource requests are configured (please check [RA2](https://cntt.readthedocs.io/en/latest/ref_arch/kubernetes/) for additional details).
+Examples, node configurations specified as: B, B.low-latency, H,  and H.very-high-speed-network.very-low-latency-edge.
 
-<a name="Table4-13"></a>
+A workload needs to specify the configuration and capabilities of the infratsructure that it can un on, the size of the resources it needs, and additional information (extra-specs) such as whether the workload can share core siblings (SMT thread) or not, whether it has affinity (viz., needs to be placed on the same infrastructure node) with other workloads, etc. The capabilities required by the workload can, thus, be specified as:
 
+>  \<profile name>[.\<profile_extension>][.\<extra profile specs>].\<workload flavour specs>[.\<extra-specs>]
 
+where the \<workload flavour specs> are specified as defined in [4.2.4.3 Workload Flavours and Other Capabilities Specifications Format](#4.2.4.3) below.
 
+<a name="4.2.1"></a>
+### 4.2.1 Profiles
+
+<a name="4.2.1.1"></a>
+#### 4.2.1.1 Basic Profile
+
+Hardware resources configured as per the Basic profile (B) such that they are only suited for workloads that tolerate variable performance, including latency, and resource over-subscription. Only Simultaneous Multi-Threading (SMT) is configured on nodes supporting the Basic profile. With no NUMA alignment, the vCPUs executing processes may not be on the same NUMA node as the memory used by these processes. When the vCPU and memory are on different NUMA nodes, memory accesses are not local to the vCPU node and thus add latency to memory accesses. The Basic profile supports over subscription (using CPU Allocation Ratio) which is specified as part of sizing information in the workload profiles.
+
+<a name="4.2.1.2"></a>
+#### 4.2.1.2 High Performance Profile
+
+The high-performance profile (H) is intended to be used for workloads that require predictable performance, high network throughput requirements and/or low network latency. To satisfy predictable performance needs, NUMA alignment, CPU pinning, and Huge pages are enabled. For obvious reasons, the high-performance profile doesn’t support over-subscription.
 
 <a name="4.2.2"></a>
-### 4.2.2 Virtual Network Interface Specifications
+### 4.2.2 Profiles Specifications & Capability Mapping
+
+| Ref | Capability  | Basic | High Performance | Notes |
+|-----|---------|----------|----------|--------|
+| e.cap.006 | CPU pinning | No | Yes | Exposed performance capabilities as per Table 4-2. |
+| e.cap.007 | NUMA alignment  | No | Yes | |
+| e.cap.013 | SR-IOV over PCI-PT  | No   | Yes | |
+| i.cap.018 | Huge page support  | No  | Yes | Internal performance capabilities as per Table 4-7. |
+| e.cap.018 | Simultaneous Multithreading (SMT) | Yes | Yes | |
+| e.cap.019 | vSwitch Optimisation (DPDK) | No | Yes| |
```

Comment:
@pgoyal01 e.cap.019 "vSwitch optimisation " must be moved to profile extension,same argument as for SR-IOV

Reply:
@karinesevilla we need some level of network performance. Can we add a note to the effect that "DPDK doesn't have to be used is some other network acceleration method is being utilised" - or something to that effect.

Reply:
@pgoyal01 yes, I think so

Reply:
@karinesevilla Done - Thanks